tests/reactive_fluid_transport_no_reaction.prm

```diff
@@ -0,0 +1,144 @@
+# This model tests the reactive fluid transport material
+# 'no reaction' scenario to ensure that there is no reaction
+# between the fluid and solid phases. The values of
+# porosity (0) and the bound fluid (0.01) are initially
+# spatially uniform. The absence of reactions between
+# the two phases and near zero velocities (free-slip on
+# all sides) produces no change in their values through time.
```

Comment:
This is very similar to the zero solubility test below. Can you include the other test here with
include $ASPECT_SOURCE_DIR/tests/reactive_fluid_transport_zero_solubility.prm
and the only list the parameters that are different? This way you do not have to duplicate everything.